pkg/envoy/npds.pb.go

```diff
@@ -0,0 +1,479 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
+// source: npds.proto
+
+package envoy
+
+import proto "github.com/golang/protobuf/proto"
+import fmt "fmt"
+import math "math"
+import envoy_api_v2 "api"
```

Comment:
@jrajahalme The hack we're doing in `envoy/Makefile` to compile data-plane-api's .proto files into a package/directory name different from the .proto files' package name is not sufficient anymore. Any protocol buffer which imports any of those .proto files will not compile.
For instance, in this case, `npds.proto` imports these files from data-plane-api:
```
import "api/address.proto";
import "api/discovery.proto";
import "api/rds.proto";
```
And here are the resulting imports in the generated `npds.pb.go`:
```
import envoy_api_v2 "api"
import envoy_api_v22 "api"
import envoy_api_v25 "api"
```
This doesn't compile. For this to work, the imports should be:
```
import "github.com/cilium/cilium/pkg/envoy/api"
```

I can think of 2 solutions:
1. If we really wanted to keep the .pb.go files generated from data-plane-api into the wrong package directory (different from the `go_package` option in the .proto files), we would have to rewrite those .proto files to modify the `go_package` options in every one of those files. But I think this may have nasty consequences on protobuf encoding.
1. Use the exact same `option go_package = "api";` in my own `npds.proto` and `nphds.proto`, and generate the .pb.proto files into the same package/directory (`pkg/envoy/api`).

I will try 2. since it's the least invasive solution.

In any case, this will need some work when rebasing from data-plane-api's master since they moved the proto files into Go package `envoy.api.v2`.

Any thoughts?